packages/card-picker/tests/dummy/config/optional-features.json

```diff
@@ -1,3 +1,3 @@
 {
-  "jquery-integration": false
+  "jquery-integration": true
```

Comment:
why do we need jQuery for this?

Reply:
This is because we need ember-data and it's still needed for now 👍